core/src/test/scala/org/apache/spark/deploy/ExternalShuffleServiceSuite.scala

```diff
@@ -131,8 +106,6 @@ class ExternalShuffleServiceSuite extends ShuffleSuite with BeforeAndAfterAll wi
       val bms = eventually(timeout(2.seconds), interval(100.milliseconds)) {
         val locations = sc.env.blockManager.master.getLocations(blockId)
         assert(locations.size === 2)
-        assert(locations.map(_.port).contains(server.getPort),
-          "external shuffle service port should be contained")
```

Comment:
Instead of dropping this, I guess we could check whether it's one of the two ESS ports?

Reply:
It's just a bit hard to get the port. And the test itself verifies the external shuffle service works in the end.  So I think it might not be worth to add more changes here.